packages/kbn-search-index-documents/utils/fetch_with_pagination.ts

```diff
@@ -1,13 +1,13 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
 
 import { SearchHit, SearchResponse, SearchTotalHits } from '@elastic/elasticsearch/lib/api/types';
-
-import { Paginate } from '../../common/types/pagination';
+import { Paginate } from '../types';
```

Comment:
I moved this file from enterprise search plugin & this one apparently wasn't being used by any other components in  enterprise_search. If we want this  in enterprise_search, I can keep a copy there